README.md

```diff
@@ -133,4 +140,4 @@ WebHostingHub Glyphs
 - License: http://creativecommons.org/licenses/by/3.0/
 
 Splashy Icons (free as in free) 
-- Web page: http://splashyfish.com/icons/
```

Comment:
Please see comments on PR# 2003 regarding the proposed changes to the content of this README file. It would be nice to have this PR re-submitted without the README changes so we can take advantage of the rest of the corrections while we consider the README changes. Thanks!